snippets/card-product.liquid

```diff
@@ -11,6 +11,7 @@
     - lazy_load: {Boolean} Image should be lazy loaded. Default: true (optional)
     - show_quick_add: {Boolean} Show the quick add button.
     - section_id: {String} The ID of the section that contains this card.
+    - wrapper_class: {String} Wrapper class for card Default: "" (optional)
```

Comment:
Are we still using `wrapper_class` on `card-product`? I don't see it being used anywhere.